indico/modules/events/surveys/templates/survey_results.html

```diff
@@ -5,5 +5,146 @@
+                        {{ single_choice_answer(question) }}
+                    {% elif question.field_type == 'multiselect' %}
+                        {{ multiple_choice_answer(question) }}
+                    {% endif %}
+                </div>
+            </div>
+        {% endfor %}
+    </div>
 {% endblock %}
+
+
+{% macro text_answer(question) %}
+    {% for answer in question.get_summary() %}
+        <div class="text-answer">{{ answer }}</div>
+    {% endfor %}
+{% endmacro %}
+
+
+{% macro number_answer(question) %}
+    {% set summary = question.get_summary() %}
+    {% if summary %}
+        <div class="number-answer">
+            <div class="i-badge">
+                <div class="i-badge-content">
+                    <div class="i-badge-value">
+                        {{ summary.min }}
+                    </div>
+                    <span class="i-badge-title">
+                        {% trans %}Min{% endtrans %}
+                    </span>
+                </div>
+            </div>
+            <div class="i-badge">
+                <div class="i-badge-content">
+                    <div class="i-badge-value">
+                        {{ '%0.2f'|format(summary.average) }}
+                    </div>
+                    <span class="i-badge-title">
+                        {% trans %}Average{% endtrans %}
+                    </span>
+                </div>
+            </div>
+            <div class="i-badge">
+                <div class="i-badge-content">
+                    <div class="i-badge-value">
+                        {{ summary.max }}
+                    </div>
+                    <span class="i-badge-title">
+                        {% trans %}Max{% endtrans %}
+                    </span>
+                </div>
+            </div>
+            <div class="i-badge">
+                <div class="i-badge-content">
+                    <div class="i-badge-value">
+                        {{ summary.total }}
+                    </div>
+                    <span class="i-badge-title">
+                        {% trans %}Total{% endtrans %}
+                    </span>
+                </div>
+            </div>
+        </div>
+    {% endif %}
+{% endmacro %}
+
+
+{% macro single_choice_answer(question) %}
+    {{ _choice_answer(question) }}
+    {% set summary = question.get_summary() %}
+    <script>
+        var chart = new Chartist.Pie('#q-{{ question.id }}-chart', {
+            labels: {{ summary.relative.keys()|tojson }},
+            series: {{ summary.relative.values()|tojson }}
+        }, {
+            donut: true,
+            donutWidth: 20,
+            startAngle: 270,
+            total: 1,
+            chartPadding: 20,
+            labelOffset: 20,
+            labelDirection: 'explode'
+        }).on('draw', function(data) {
+            if (data.value === 0) {
+                chart.data.labels[data.index] = ''
```

Comment:
I'm moving the generation of the charts to `surveys.js` and it's already fixed locally.